_data/projects/manipulation/uproot-browser.yml

```diff
@@ -0,0 +1,10 @@
+name: uproot-browser
+projlogo: /assets/images/projlogo/logo_uproot_browser.png
+description: Terminal browser and tools for uproot files
```

Comment:
```suggestion
description: Terminal browser and tools for ROOT files
```

Reply:
If you fancy something longer you could say "uproot-engined* or "based on uproot", but the files are ROOT files.